lib/generate-sitemap.js

```diff
@@ -16,18 +16,17 @@ function generateSitemap(batfishConfig) {
     path.join(batfishConfig.outputDirectory, 'sitemap.xml')
   );
   return new Promise((resolve, reject) => {
-    sitemapStatic(
-      sitemapWriter,
-      {
+    try {
```

Comment:
That makes sense, done!

Is there a reason you used the anonymous function wrapper in your example? Just curious, I wrote it as follows, but I can change it!
```
sitemapWriter.on('error', reject);
sitemapWriter.on('finish', resolve);
```